packages/blocks/src/api/parser.js

```diff
@@ -404,8 +412,13 @@ export function getMigratedBlock( block, parsedAttributes ) {
  */
 export function createBlockWithFallback( blockNode ) {
 	const { blockName: originalName } = blockNode;
+	// The tripartite structure of the block type includes its attributes, inner
```

Comment:
"Tripartite" is a very obscure word; perhaps this should be simplified to "The structure of a block type includes its attributes, inner blocks, and inner HTML."